src/test/java/net/sf/jabref/gui/journals/ManageJournalAbbreviationsTest.java

```diff
@@ -0,0 +1,560 @@
+        int size = 5;
+        Abbreviation testAbbreviation = new Abbreviation("JabRefTestEntry", "JTE");
+        editAbbreviation(testAbbreviation);
+        Assert.assertEquals(size, viewModel.abbreviationsProperty().size());
+        Assert.assertTrue(viewModel.abbreviationsProperty().contains(new AbbreviationViewModel(testAbbreviation)));
+
+        viewModel.addNewFile(testFile5EntriesWithDuplicate);
+        selectLastJournalFile();
+        selectLastAbbreviation();
+        size = viewModel.abbreviationsProperty().size();
+        viewModel.deleteAbbreviation();
+        Abbreviation testAbbreviation1 = new Abbreviation("SomeOtherEntry", "SOE");
+        addAbbrevaition(testAbbreviation1);
+
+        Assert.assertEquals(size, viewModel.abbreviationsProperty().size());
+        Assert.assertTrue(viewModel.abbreviationsProperty().contains(new AbbreviationViewModel(testAbbreviation1)));
+
+        viewModel.saveJournalAbbreviationFiles();
+
+        String expected = "Abbreviations = Abb" + NEWLINE + "Test Entry = TE" + NEWLINE + "MoreEntries = ME" + NEWLINE
+                + "JabRefTestEntry = JTE" + NEWLINE + "";
+        String actual = Files.contentOf(new File(testFile4Entries), StandardCharsets.UTF_8);
+        Assert.assertEquals(expected, actual);
+
+        expected = "Abbreviations = Abb" + NEWLINE + "Test Entry = TE" + NEWLINE + "MoreEntries = ME" + NEWLINE
+                + "SomeOtherEntry = SOE" + NEWLINE + "";
+        actual = Files.contentOf(new File(testFile5EntriesWithDuplicate), StandardCharsets.UTF_8);
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void testSaveExternalFilesListToPreferences() throws DuplicatedJournalFileException {
+        saveFilesToPreferences();
+
+        List<String> expected = Arrays.asList(testFile1Entries, testFile3Entries, testFile4Entries,
+                testFile5EntriesWithDuplicate);
+        List<String> actual = Globals.prefs.getStringList(JabRefPreferences.EXTERNAL_JOURNAL_LISTS);
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void testBindings() throws FileNotFoundException {
+        AbbreviationsFileViewModel testFile = new AbbreviationsFileViewModel(testFile5EntriesWithDuplicate);
+        testFile.readAbbreviations();
+        SimpleListProperty<AbbreviationsFileViewModel> testProperty = new SimpleListProperty<>(
+                FXCollections.observableArrayList(testFile));
+
+        viewModel.bindFileItems(testProperty);
+
+        Assert.assertEquals(testFile, viewModel.currentFileProperty().get());
+    }
+
+    private String createTemporaryTestFile(String name, String content) {
+        File testFile = null;
+        try {
+            testFile = tempFolder.newFile(name);
+            try (OutputStream outputStream = new FileOutputStream(testFile)) {
+                outputStream.write(content.getBytes(StandardCharsets.UTF_8));
+            }
+        } catch (IOException e) {
```

Comment:
Don't catch this exception here. If something goes wrong with the initialization, the tests should better fail.